sdk/messaging/azservicebus/go.mod

```diff
@@ -3,17 +3,12 @@ module github.com/Azure/azure-sdk-for-go/sdk/messaging/azservicebus
 go 1.16
 
 require (
-	github.com/Azure/azure-amqp-common-go/v3 v3.2.1
-	github.com/Azure/azure-sdk-for-go v51.1.0+incompatible
 	github.com/Azure/azure-sdk-for-go/sdk/azcore v0.20.0
 	github.com/Azure/azure-sdk-for-go/sdk/azidentity v0.12.0
-	github.com/Azure/azure-sdk-for-go/sdk/internal v0.8.1
+	github.com/Azure/azure-sdk-for-go/sdk/internal v0.8.2
+	github.com/Azure/azure-sdk-for-go/sdk/messaging/internal v0.0.0-20211118043544-abfa4f313a9c
 	github.com/Azure/go-amqp v0.16.4
 	github.com/Azure/go-autorest/autorest v0.11.18
```

Comment:
This is also a track 1 dependency.

Reply:
Oh, it's GONE now! Look at that beautiful and svelte go.mod!